webapp/app/model/components/__init__.py

```diff
@@ -199,3 +199,7 @@ class InfoTaxReturnForPensionersProps(ComponentPlausibleProps):
 
 class AmbassadorInfoMaterialProps(ComponentPlausibleProps):
     pass
+
+
+class KrankheitsKostenInfoPageProps(ComponentProps):
```

Comment:
Maybe better in English? We agreed to use German if we have very specific terms like "Freischaltcode". What about "MedicalExpensesInfoPageProps"?